.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Nightly End-to-end Tests
+
+on:
+  schedule:
```

Comment:
Can't we just also include a `schedule` in the existing e2e test workflow?

Reply:
Done (with the caveat of duplicated code, Since we can't have a conditional on the runs-on everything after it has to be duplicated)